pipeline_steps/common.groovy

```diff
@@ -106,7 +106,7 @@ def venvPlaybook(Map args){
       sh """#!/bin/bash -x
         which scl && source /opt/rh/python27/enable
         set +x; . ${env.WORKSPACE}/.venv/bin/activate; set -x
-        export ANSIBLE_HOST_KEY_CHECKING=False
+        export ANSIBLE_CONFIG=${env.WORKSPACE}/rpc-gating/ansible.cfg
```

Comment:
This path doesn't match the location of the file. Either move the file to root of the repo or add `/playbooks/` here.